examples/com/amazonaws/examples/EncryptionContextOverridesWithDynamoDBMapper.java

```diff
@@ -48,107 +48,73 @@ public static void main(String[] args) throws GeneralSecurityException {
     }
 
     public static void encryptRecord(final String cmkArn,
-                                     final String newEncryptionContextTableName,
                                      AmazonDynamoDB ddb,
                                      AWSKMS kms) throws GeneralSecurityException {
-        // Sample object to be encrypted
-        ExampleItem record = new ExampleItem();
-        record.setPartitionAttribute("is this");
-        record.setSortAttribute(55);
-        record.setExample("my data");
+        final String dynamoTableName = "DuckTable";
+        final String dynamoBackupName = "DuckBackup";
+
+        DuckPojo duckPojo = new DuckPojo();
+        duckPojo.setDuckName("Quackley");
+        duckPojo.setSwimming(true);
 
-        // Set up our configuration and clients
         final DirectKmsMaterialProvider cmp = new DirectKmsMaterialProvider(kms, cmkArn);
         final DynamoDBEncryptor encryptor = DynamoDBEncryptor.getInstance(cmp);
 
-        Map<String, String> tableNameEncryptionContextOverrides = new HashMap<>();
```

Comment:
If this is the only example we have for table name overrides, can we include this as, say, a commented describing or at least referencing the `overrideEncryptionContextTableNameUsingMap` option?

Reply:
Yes, that sounds like a good idea.

Reply:
Added a comment to mention `overrideEncryptionContextTableNameUsingMap`